LEGO1/islepathactor.cpp

```diff
@@ -0,0 +1,21 @@
+#include "islepathactor.h"
+
+// OFFSET: LEGO1 0x10001e60 STUB
+IslePathActor::IslePathActor()
```

Comment:
`Helicopter::Helicopter` is at this address.
You probably mean 0x1001a200
```suggestion
// OFFSET: LEGO1 0x1001a200 STUB
IslePathActor::IslePathActor()
```